apps/portals-base/package.json

```diff
@@ -33,26 +32,12 @@
   "scripts": {
     "clean": "rimraf coverage",
     "lint": "eslint src",
-    "copy-assets": "cp src/config/style/favicon* public/; cp src/config/style/logo.svg public/; cp src/config/style/socialmedia.png public/",
-    "save-build-date": "date > public/build-date.txt",
-    "start": "vite",
-    "build": "pnpm copy-assets; pnpm save-build-date; vite build",
```

Comment:
No need to build, we will set up each portal's vite config to just look at the source of this project.